### PR TITLE
Create type builtin function

### LINKED ISSUE #186: Create `type` builtin function
To ease the conversion of Lua code to Pallene, we might want to create a `type` builtin function, that receives an `any` and returns a `string`.

```
    function is_boolean(x: any)
        return type(x) == "boolean"
    end
```

### PR DESCRIPTION
**Pull request by mambrozio**
Fix #186

### CODE REVIEW

File: pallene/builtins.lua [hunk 1]
```diff
@@ -15,6 +15,7 @@ for lua_name, typ in pairs({
     ["string.char"]  = T.Function({T.Integer()}, {T.String()}),
     ["string.sub"]   = T.Function({T.String(), T.Integer(), T.Integer()}, {T.String()}),
     ["tofloat"]      = T.Function({T.Integer()}, {T.Float()}),
+    ["btype"]         = T.Function({T.Any()}, {T.String()}),
```
Comment (hugomg) [inline pallene/builtins.lua]: Will it be possible to call this function "type" instead of "btype"?

Comment (gligneul) [reply pallene/builtins.lua]: I would like it to be called `type` as well.

Comment (gligneul) [reply pallene/builtins.lua]: There is no problem renaming the current `type` to `typealias` or even `alias`.

Comment (hugomg) [reply pallene/builtins.lua]: Agreed. I would prefer renaming the current `type` to `typealias`.

Comment (mambrozio) [reply pallene/builtins.lua]: Changed to `typealias` in the new commit.

**Comment by mambrozio** [comment]
Updated manual with the last commit.

**Pull request merged**